### PR TITLE
Add custom Groovy lexer to sphinx/rtd

### PR DESCRIPTION
**Pull request by elevans**
This PR adds a `lexers.py` module that contains a custom Groovy lexer that adds "#@" lines as comments. To use this custom lexer simply define your code block like so:

```text
.. code-block:: scijava-groovy
```

This PR resolves the annoying errors with "#@" in groovy scripts and stops the error highlighting in our scijava ops docs.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -56,6 +59,13 @@
 myst_heading_anchors = 4
 nb_execution_mode = "off"
 
+# -- Custom Lexers -----------------------------------------------------------
+
+from lexers import FijiGroovyLexer
```
Comment (ctrueden) [inline docs/conf.py]: Could we please call it `SciJavaGroovyLexer` instead? It really has nothing to do with Fiji specifically. The `#@` syntax is part of the SciJava scripting subsystem.

File: docs/conf.py [hunk 2]
```diff
@@ -56,6 +59,13 @@
...
+
+def setup(app):
+    app.add_lexer('fijigroovy', FijiGroovyLexer)
```
Comment (ctrueden) [inline docs/conf.py]: And similarly, here I would name the lexer `scijava-groovy`. With dash for clarity, assuming dashes are allowed. Or else `scijava_groovy` or `sciJavaGroovy` if dashes cannot be used.

**Review by ctrueden — CHANGES_REQUESTED** [review]
Thanks a lot for figuring this out! I would just request the name be "SciJava" instead of "Fiji" since that's the relevant layer here.

File: docs/conf.py (hunk 1 above)
Comment (elevans) [reply docs/conf.py]: Done!

File: docs/conf.py (hunk 2 above)
Comment (elevans) [reply docs/conf.py]: We can use dashes. The SciJava Groovy lexer alias is now `scijava-groovy`.

**Review by elevans — COMMENTED** [review]
Thanks for the feedback @ctrueden! We can use this custom Groovy lexer with the `scijava-groovy` alias.

**Review by ctrueden — APPROVED** [review]
Thanks @elevans! LGTM

**Pull request merged**